include/teakra/teakra.h

```diff
@@ -72,15 +71,17 @@ class Teakra {
     void AHBMWrite32(std::uint32_t addr, std::uint32_t value);
 
     // core
-    void Run(unsigned cycle);
+    unsigned Run(unsigned cycle);
```

Comment:
s/unsigned/u32 probably just like Processor::Run

Reply:
Given this an interface file, it would be preferred to use `std::uint32_t` instead of `u32`

Reply:
Seconded